lib/katello/tasks/upgrades/2.4/import_rpms.rake

```diff
@@ -0,0 +1,12 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '2.1' do
```

Comment:
Also need to include this in the engine so it gets loaded and open a PR to the installer to include this